.env

```diff
@@ -1 +1,2 @@
 SASS_PATH="node_modules"
+REACT_APP_GITHUB_PERSONAL_ACCESS_TOKEN=4c3fa2c2490a3ec64125ee3809e0c36e3c34b37c
```

Comment:
You should not commit your personal access token.

Reply:
Hi,

I already moved the personal access token to .env.local. Please check it again. Thanks.